angular/core/components/thread_page/comment_form/comment_form.coffee

```diff
@@ -5,6 +5,11 @@ angular.module('loomioApp').directive 'commentForm', ->
   replace: true
   controller: ($scope, $rootScope, FormService, Records, Session, KeyEventService, AbilityService, MentionService, AttachmentService, ScrollService, EmojiService, ModalService, SignInForm) ->
 
+    $scope.$on 'remindUndecided', (event, undecided) ->
+      undecided = _.flatten _.values(undecided)
+      undecided = _.map undecided, (member) -> "@" + "#{member}"
```

Comment:
What are your thoughts on, instead of broadcasting all those undecided members, doing something like

```
return unless $scope.discussion.activeProposal()
_.map $scope.discussion.activeProposal().undecidedMembers(), (member) -> "@#{member.username}"
```